website/layouts/language.erb

```diff
@@ -0,0 +1,1030 @@
+<% wrap_layout :inner do %>
+  <% content_for :sidebar do %>
+    <h4><a href="/docs/configuration/index.html">Terraform Language</a></h4>
+
+    <ul class="nav docs-sidenav">
+
+      <li>
+        <a href="/docs/configuration/index.html">Home</a>
+      </li>
+
+      <li>
+        <a href="#">Introduction</a>
+        <ul class="nav">
+          <li>
+            <a href="/docs/configuration/overview.html">Language Overview</a>
```

Comment:
Cf. my comment above. Most of the overview pages are overviews of what's available in a given section, but this one gives you an overview of the language. Open to suggestions on this! IMO, simply going with "overview" would make this harder to navigate, because it wouldn't communicate what's in here.

Reply:
Sorry its taken me so long to respond to this one. I fell down a bit of a hole trying to figure out what to call that page. 

Some totally conflicting thoughts: 
- Because Language overview is one level separated from the overview maybe its ok that their names are similar? 
- If we can't think of a more specific title for the Language overview page maybe the page itself needs work? Language features, components? What unifies the information on this page? 
- Maybe we should break the convention of stub pages and make the Language overview the content for the overview page? If the content in that docs section doesn't match the content in the language overview then maybe we need to do more work on the overview page or docs section itself to make them match?